extensions/interactions/legacy_html_field_types_to_rule_specs/html_field_types_to_rule_specs_stateV41.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Here and for the next file, let's preserve the tradition of all-lowercase filenames -- maybe legacy_html_field_types_to_rule_specs/state_version_41/html_field_types_to_rule_specs_v41.json or 
legacy_html_field_types_to_rule_specs_by_state_version/html_field_types_to_rule_specs_v41.json